starboard/build/platform_configuration.py

```diff
@@ -177,3 +177,11 @@ def GetTestTargets(self):
         'slot_management_test',
         'starboard_platform_tests',
     ]
+
+  def GetTestVirtualTargets(self):
+    """Gets all tests to be run in a black box test run.
```

Comment:
Shall we unify the description of `GetTestVirtualTargets` across different overrides?

Reply:
Renamed to GetTestBlackBoxTargets and similar for other occasions.